utils/python/CIME/SystemTests/seq.py

```diff
@@ -14,7 +14,59 @@ def __init__(self, caseroot, case):
         """
         initialize an object interface to file env_test.xml in the case directory
         """
-        SystemTestsCommon.__init__(self, caseroot, case)
+        SystemTestsCommon.__init__(self, caseroot, case, expected=["TEST"])
```

Comment:
See my comments in the slack general.   It causes a comparison of env_run.xml with the original to reset this field to the original value.